templates/layout.html

```diff
@@ -10,7 +10,7 @@
   <!--Import Google Font-->
   <link href="https://fonts.googleapis.com/css?family=Roboto|Roboto+Slab&display=swap" rel="stylesheet">
   <link href="{{ url_for('static', filename='stylesheets/main.scss.css') }}" rel="stylesheet" type="text/css">
-<link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
```

Comment:
[nit] This should be moved up one with the other Google font. We may want to add a comment for the link above to distinguish it (ex: `<!-- Custom CSS -->`) from the others.

Reply:
no objections from me

Reply:
moved!